### DISCUSSION TITLE
Inverter & Losses

### DISCUSSION THREAD

**Discussion by davidrapan** [davidrapan/ha-solarman#247] [Ideas]
#### Did you know you can add an `Inverter Total Losses` sensor into your energy dashboard as an individual device? 😉
![energy_dashboard_total_losses](https://github.com/user-attachments/assets/9bf98442-a7ae-485e-a0bc-2a9f3fcd1ae3)

**Comment by davidrapan** [comment]
@PV-Joe, hm? 😉

**Reply by PV-Joe** [reply]
Thx for the hint but I've already added it in the energy dashboard 😄👍

**Reply by davidrapan** [reply]
Yeah I was wondering if it came to your mind or not! 😆
